challenge_2/rust/josephpd3/src/lib.rs

```diff
@@ -0,0 +1,33 @@
+use std::collections::HashMap;
+
+pub fn get_lonely_items(v: &[char]) -> Vec<char> {
+    let mut lonely = vec![];
+    let mut counts = HashMap::new();
+
+    for ch in v {
+        let mut present = true; // Flags are fun!
```

Comment:
Even more fun than flags are Rust's error chaining. I'm reviewing this from the airport so I can't fact-check myself, but would a `Result` returning accessor like `.get(ch).or_insert(ch, 0)` work here?